cmd/minishift/cmd/start_preflight.go

```diff
@@ -44,8 +44,9 @@ import (
 )
 
 const (
-	StorageDisk   = "/mnt/?da1"
-	GithubAddress = "https://github.com"
+	StorageDisk                  = "/mnt/?da1"
+	GithubAddress                = "https://github.com"
+	hypervDefaultVirtualSwitchId = "c08cb7b8-9b3c-408e-8e30-5e16a3aeb444"
```

Comment:
Can we put a comment here and link it to the https://blogs.technet.microsoft.com/virtualization/2017/11/13/whats-new-in-hyper-v-for-windows-10-fall-creators-update/ or any better location which state it is a constant and not going to change for system by system?

Reply:
I will look in technet... but so far I hadn't found...

Reply:
Love to have comment here.

Reply:
how is a comment going to help ? The actual variable name should describe it...